sections/image-with-text.liquid

```diff
@@ -159,7 +159,7 @@
       "settings": [
         {
           "type": "text",
-          "id": "button_label",
+          "id": "text",
```

Comment:
This change is not required either as only one block of each type can be added which should suffice

<img width="366" alt="Screen Shot 2021-08-11 at 8 41 41 AM" src="https://user-images.githubusercontent.com/22102095/129032213-8a42740f-a104-41ee-bc13-5bd2a6f9999c.png">

Reply:
Buttons can behave unexpectedly, especially if we have more than one button in a block. Let's keep button blocks static named. We'll improve their iconography to reflect the type better later.